app/controllers/state_file/questions/nj_medical_expenses_controller.rb

```diff
@@ -0,0 +1,9 @@
+module StateFile
+  module Questions
+    class NjMedicalExpensesController < QuestionsController
+      include ReturnToReviewConcern
+
+      before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
```

Comment:
[DUST] Is @filing_year being used at all in the page?

Reply:
Nvm I see it is used on line 18 of [app/views/state_file/questions/nj_medical_expenses/edit.html.erb](https://github.com/codeforamerica/vita-min/pull/4869/files#diff-260d83dc4b3add3a84c8feaab5e058bc20cb28d5f653465bb41406f32e1195fe)